From 6323f5f4c056c84d3beaee1f1057ade437cf1e8f Mon Sep 17 00:00:00 2001
From: Doug Goldstein <cardoe@gentoo.org>
Date: Thu, 4 Feb 2010 00:37:42 -0600
Subject: [PATCH] virt-pki-validate contains unexpanded SYSCONFDIR variable

virt-pki-validate contains an unexpanded SYSCONFDIR variable which
causes it not to function at all.
---
 tools/virt-pki-validate.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/tools/virt-pki-validate.in b/tools/virt-pki-validate.in
index a428a96..3e8c642 100755
--- a/tools/virt-pki-validate.in
+++ b/tools/virt-pki-validate.in
@@ -25,7 +25,7 @@ echo Found $CERTOOL
 #
 # Check the directory structure
 #
-PKI="$(SYSCONFDIR)/pki"
+PKI="@SYSCONFDIR@/pki"
 if [ ! -d $PKI ]
 then
     echo the $PKI directory is missing, it is usually
-- 
1.6.6.1

